docs/API.md

```diff
@@ -21,7 +21,7 @@ it('can handle commands synchronously', () => {
 you need return the command promise so it gets resolved properly as well as access the value when the promise got resolve:
 
 ```js
-it('handles commands as promises', ()  =>{
+it('handles commands as promises', ()  => {
```

Comment:
```suggestion
it('handles commands as promises', () => {
```

while we at it 😉

Reply:
hm, I will check whether exists or not eslint rule for it :)

Reply:
It is docs. Linter doesn't process `.md` files

Reply:
Fixed!